src/CMakeLists.txt

```diff
@@ -35,7 +35,9 @@ if (libzmq_FOUND)
   set(SOURCE_FILES ${SOURCE_FILES} binpickingtaskzmq.h binpickingtaskzmq.cpp mujinzmq.cpp)
 endif()
 
-add_library(libmujincontrollerclient SHARED ${SOURCE_FILES})
+add_library(libmujincontrollerclientbase OBJECT ${SOURCE_FILES})
```

Comment:
Out of curiosity: is this an optimization since we're making `SHARED` and `STATIC` objects separately otherwise?

Reply:
right, actually https://github.com/mujin/controllerclientcpp/pull/150 covers this